Gemfile

```
@@ -3,7 +3,7 @@ source 'https://rubygems.org'
 ruby '2.5.7'
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '~> 5.2.4.4'
+gem 'rails', '6.0.1'
```

I'd recommend to update to the last patch version which is 6.0.3.5 in this case.

```suggestion
gem 'rails', '~> 6.0.3.5'
```